openedx_ledger/models.py

```diff
@@ -287,7 +287,7 @@ class Meta:
             "The globally unique content identifier.  Joinable with ContentMetadata.content_key in enterprise-catalog."
         )
     )
-    reference_id = models.CharField(
+    platform_fulfillment_id = models.CharField(
```

Comment:
We discussed during standup calling this `enterprise_fulfillment_uuid` to help eliminate doubt that it points to a UUID in a table in edx-enterprise that contains "Enterprise" in the name.  Thoughts?